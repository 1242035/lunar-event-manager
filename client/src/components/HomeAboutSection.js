import React from 'react';

/**
 * Contains details on the business and its goals.
 * @param{Object} props - passed properties
 * @return{ReactElement} - react element
 */
const HomeAboutSection = () => (
  <div id="about" className="io-about">
    <h1>ABOUT US </h1>
    <p>
      <br />We are a medium-sized centre renting and event management company situated in Lagos.
      <br />All our centres offer flexible space for a range of events. We are able arrange events
      <br />that thousands of guests. Our centers are fitted with state-of-the-art facilities to
      <br />make your dream event a reality.
      <br />
      <br />We also provide on-ground services like catering and traffic management in case such
      <br />services are required.
      <br />
      <br />We have the perfect venues for:
      <br />• Conferences and corporate events
      <br />• Meetings
      <br />• Private and fine dining
      <br />• Gala dinners
      <br />• Press conferences
      <br />• Lectures and seminars (two theatres with tiered seating)
      <br />• Wedding Venue for ceremonies and receptions
      <br />• Fashion shows
      <br />• Exhibitions – small and medium
      <br />• Filming location for feature films or television series
    </p>
  </div>
);

export default HomeAboutSection;
