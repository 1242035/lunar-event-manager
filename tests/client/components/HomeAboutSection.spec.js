import React from 'react';
import { shallow } from 'enzyme';
import HomeAboutSection from '../../../client/src/components/HomeAboutSection';

describe('HomeAboutSection', () => {
  let wrapper;

  it('Should maintain existing snapshot', () => {
    wrapper = shallow(<HomeAboutSection />);
    expect(wrapper).toMatchSnapshot();
  });

  it('Should render component', () => {
    wrapper = shallow(<HomeAboutSection />);
    expect(wrapper.length).toEqual(1);
  });
});
