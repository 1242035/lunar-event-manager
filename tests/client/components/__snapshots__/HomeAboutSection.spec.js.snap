// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomeAboutSection Should maintain existing snapshot 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <HomeAboutSection />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <h1>
          ABOUT US 
        </h1>,
        <p>
          <br />
          We are a medium-sized centre renting and event management company situated in Lagos.
          <br />
          All our centres offer flexible space for a range of events. We are able arrange events
          <br />
          that thousands of guests. Our centers are fitted with state-of-the-art facilities to
          <br />
          make your dream event a reality.
          <br />
          <br />
          We also provide on-ground services like catering and traffic management in case such
          <br />
          services are required.
          <br />
          <br />
          We have the perfect venues for:
          <br />
          • Conferences and corporate events
          <br />
          • Meetings
          <br />
          • Private and fine dining
          <br />
          • Gala dinners
          <br />
          • Press conferences
          <br />
          • Lectures and seminars (two theatres with tiered seating)
          <br />
          • Wedding Venue for ceremonies and receptions
          <br />
          • Fashion shows
          <br />
          • Exhibitions – small and medium
          <br />
          • Filming location for feature films or television series
        </p>,
      ],
      "className": "io-about",
      "id": "about",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "ABOUT US ",
        },
        "ref": null,
        "rendered": "ABOUT US ",
        "type": "h1",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <br />,
            "We are a medium-sized centre renting and event management company situated in Lagos.",
            <br />,
            "All our centres offer flexible space for a range of events. We are able arrange events",
            <br />,
            "that thousands of guests. Our centers are fitted with state-of-the-art facilities to",
            <br />,
            "make your dream event a reality.",
            <br />,
            <br />,
            "We also provide on-ground services like catering and traffic management in case such",
            <br />,
            "services are required.",
            <br />,
            <br />,
            "We have the perfect venues for:",
            <br />,
            "• Conferences and corporate events",
            <br />,
            "• Meetings",
            <br />,
            "• Private and fine dining",
            <br />,
            "• Gala dinners",
            <br />,
            "• Press conferences",
            <br />,
            "• Lectures and seminars (two theatres with tiered seating)",
            <br />,
            "• Wedding Venue for ceremonies and receptions",
            <br />,
            "• Fashion shows",
            <br />,
            "• Exhibitions – small and medium",
            <br />,
            "• Filming location for feature films or television series",
          ],
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {},
            "ref": null,
            "rendered": null,
            "type": "br",
          },
          "We are a medium-sized centre renting and event management company situated in Lagos.",
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {},
            "ref": null,
            "rendered": null,
            "type": "br",
          },
          "All our centres offer flexible space for a range of events. We are able arrange events",
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {},
            "ref": null,
            "rendered": null,
            "type": "br",
          },
          "that thousands of guests. Our centers are fitted with state-of-the-art facilities to",
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {},
            "ref": null,
            "rendered": null,
            "type": "br",
          },
          "make your dream event a reality.",
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {},
            "ref": null,
            "rendered": null,
            "type": "br",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {},
            "ref": null,
            "rendered": null,
            "type": "br",
          },
          "We also provide on-ground services like catering and traffic management in case such",
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {},
            "ref": null,
            "rendered": null,
            "type": "br",
          },
          "services are required.",
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {},
            "ref": null,
            "rendered": null,
            "type": "br",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {},
            "ref": null,
            "rendered": null,
            "type": "br",
          },
          "We have the perfect venues for:",
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {},
            "ref": null,
            "rendered": null,
            "type": "br",
          },
          "• Conferences and corporate events",
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {},
            "ref": null,
            "rendered": null,
            "type": "br",
          },
          "• Meetings",
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {},
            "ref": null,
            "rendered": null,
            "type": "br",
          },
          "• Private and fine dining",
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {},
            "ref": null,
            "rendered": null,
            "type": "br",
          },
          "• Gala dinners",
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {},
            "ref": null,
            "rendered": null,
            "type": "br",
          },
          "• Press conferences",
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {},
            "ref": null,
            "rendered": null,
            "type": "br",
          },
          "• Lectures and seminars (two theatres with tiered seating)",
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {},
            "ref": null,
            "rendered": null,
            "type": "br",
          },
          "• Wedding Venue for ceremonies and receptions",
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {},
            "ref": null,
            "rendered": null,
            "type": "br",
          },
          "• Fashion shows",
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {},
            "ref": null,
            "rendered": null,
            "type": "br",
          },
          "• Exhibitions – small and medium",
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {},
            "ref": null,
            "rendered": null,
            "type": "br",
          },
          "• Filming location for feature films or television series",
        ],
        "type": "p",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <h1>
            ABOUT US 
          </h1>,
          <p>
            <br />
            We are a medium-sized centre renting and event management company situated in Lagos.
            <br />
            All our centres offer flexible space for a range of events. We are able arrange events
            <br />
            that thousands of guests. Our centers are fitted with state-of-the-art facilities to
            <br />
            make your dream event a reality.
            <br />
            <br />
            We also provide on-ground services like catering and traffic management in case such
            <br />
            services are required.
            <br />
            <br />
            We have the perfect venues for:
            <br />
            • Conferences and corporate events
            <br />
            • Meetings
            <br />
            • Private and fine dining
            <br />
            • Gala dinners
            <br />
            • Press conferences
            <br />
            • Lectures and seminars (two theatres with tiered seating)
            <br />
            • Wedding Venue for ceremonies and receptions
            <br />
            • Fashion shows
            <br />
            • Exhibitions – small and medium
            <br />
            • Filming location for feature films or television series
          </p>,
        ],
        "className": "io-about",
        "id": "about",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "ABOUT US ",
          },
          "ref": null,
          "rendered": "ABOUT US ",
          "type": "h1",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <br />,
              "We are a medium-sized centre renting and event management company situated in Lagos.",
              <br />,
              "All our centres offer flexible space for a range of events. We are able arrange events",
              <br />,
              "that thousands of guests. Our centers are fitted with state-of-the-art facilities to",
              <br />,
              "make your dream event a reality.",
              <br />,
              <br />,
              "We also provide on-ground services like catering and traffic management in case such",
              <br />,
              "services are required.",
              <br />,
              <br />,
              "We have the perfect venues for:",
              <br />,
              "• Conferences and corporate events",
              <br />,
              "• Meetings",
              <br />,
              "• Private and fine dining",
              <br />,
              "• Gala dinners",
              <br />,
              "• Press conferences",
              <br />,
              "• Lectures and seminars (two theatres with tiered seating)",
              <br />,
              "• Wedding Venue for ceremonies and receptions",
              <br />,
              "• Fashion shows",
              <br />,
              "• Exhibitions – small and medium",
              <br />,
              "• Filming location for feature films or television series",
            ],
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {},
              "ref": null,
              "rendered": null,
              "type": "br",
            },
            "We are a medium-sized centre renting and event management company situated in Lagos.",
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {},
              "ref": null,
              "rendered": null,
              "type": "br",
            },
            "All our centres offer flexible space for a range of events. We are able arrange events",
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {},
              "ref": null,
              "rendered": null,
              "type": "br",
            },
            "that thousands of guests. Our centers are fitted with state-of-the-art facilities to",
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {},
              "ref": null,
              "rendered": null,
              "type": "br",
            },
            "make your dream event a reality.",
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {},
              "ref": null,
              "rendered": null,
              "type": "br",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {},
              "ref": null,
              "rendered": null,
              "type": "br",
            },
            "We also provide on-ground services like catering and traffic management in case such",
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {},
              "ref": null,
              "rendered": null,
              "type": "br",
            },
            "services are required.",
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {},
              "ref": null,
              "rendered": null,
              "type": "br",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {},
              "ref": null,
              "rendered": null,
              "type": "br",
            },
            "We have the perfect venues for:",
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {},
              "ref": null,
              "rendered": null,
              "type": "br",
            },
            "• Conferences and corporate events",
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {},
              "ref": null,
              "rendered": null,
              "type": "br",
            },
            "• Meetings",
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {},
              "ref": null,
              "rendered": null,
              "type": "br",
            },
            "• Private and fine dining",
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {},
              "ref": null,
              "rendered": null,
              "type": "br",
            },
            "• Gala dinners",
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {},
              "ref": null,
              "rendered": null,
              "type": "br",
            },
            "• Press conferences",
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {},
              "ref": null,
              "rendered": null,
              "type": "br",
            },
            "• Lectures and seminars (two theatres with tiered seating)",
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {},
              "ref": null,
              "rendered": null,
              "type": "br",
            },
            "• Wedding Venue for ceremonies and receptions",
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {},
              "ref": null,
              "rendered": null,
              "type": "br",
            },
            "• Fashion shows",
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {},
              "ref": null,
              "rendered": null,
              "type": "br",
            },
            "• Exhibitions – small and medium",
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {},
              "ref": null,
              "rendered": null,
              "type": "br",
            },
            "• Filming location for feature films or television series",
          ],
          "type": "p",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
